localization/english/event_localization/wc_arthas_expedition_events_l_english.yml

```
@@ -0,0 +1,140 @@
+ wc_arthas_story_15_OPT_A:0 "Who was that wizard dressed in black?"
+
+ wc_arthas_story_20_DESC:0 "As we approach [andorhal.GetNameNoTier], a slight chill sets as well as an uneasy feeling. I can see it in the eyes of my men, as we walk amongst the deserted streets, devoid of any life and littered with boxes of foul grain.\n\nDeeper in the city stands a sickly [kelthuzad.GetWomanMan], yet one oozing with power, decaying corpses animated by an unholy force standing guard around [kelthuzad.GetHerHim].\n\n"We meet again", the necromancer says. "Have you come to stop me? It doesn't matter. I am but a simple pawn serving the lord [malganis.GetFirstName]. If you wish to stop the inevitable, go to [stratholme.GetNameNoTier] and despair at the sight of our great Scourge.""
+ wc_arthas_story_20_OPT_A:0 "Foul knave!"
+
+ wc_arthas_story_25_DESC:0 "Our group arrives in [hearthglen.GetNameNoTier] after a long period of marching. The necromancer's words linger in the back of my mind. As we traverse farther into the city, I make note of the soldiers training in the village square.\n\nA footman greets our group: "[arthas.GetTitledFirstName]! A monstrous army of undead warriors has appeared in the night and begun attacking villages indiscriminately! Now it is moving towards us!"\nThe preparations for the battle are already in full swing. As the soldiers begin to march I am alarmed by the sight of a shipment of crates, which bear the sigil of [andorhal.GetNameNoTier]. The soldier tries to calm us by saying it has already been distributed to the village. At that moment, the skin of the people around us begins to darken..."
+ wc_arthas_story_25_OPT_A:0 "They're limitless!"
+ 
+ wc_arthas_story_30_DESC_beggining:0 "The weary road draws to an end. In front of us lies"
+ wc_arthas_story_30_DESC_port_city:0 "the port city of [stratholme.GetNameNoTier], home of the Order of the Silver Hand."
+ wc_arthas_story_30_DESC_city:0 "the city of [stratholme.GetNameNoTier]."
+ wc_arthas_story_30_DESC_ending:0 "My scouts return from its gates with wretched news.\n\nThese people have all been infected. They may look fine now, but it is just a matter of time before they turn into the undead!\n\nMy men take positions outside the walls and await my orders. I know I will find [malganis.GetFirstName] inside. But this plague will bring destruction to the rest of my people. My actions will have dire consequences, not only for the citizens but my followers as well..."
```

The previous possessive pronouns indicates Arthas PoV. Changed it so it is compatible with other expedition members also.
```suggestion
 wc_arthas_story_30_DESC_ending:0 "My scouts return from its gates with wretched news.\n\nThese people have all been infected. They may look fine now, but it is just a matter of time before they turn into the undead!\n\nMy men take positions outside the walls and await my orders. I know I will find [malganis.GetFirstName] inside. But this plague will bring destruction to the rest of our people. Our actions will have dire consequences, not only for the citizens but our followers as well..."
```

---

As I wrote as an answer to your previous comment, I thought so at first, too, but since you're a vassal owning land (and troops) every time, I think it's not that bad. Even a count can be spoken to as mylord and can order his men to take positions, or think about the consequences for his accompanying soldiers.

---

> As I wrote as an answer to your previous comment, I thought so at first, too, but since you're a vassal owning land (and troops) every time, I think it's not that bad. Even a count can be spoken to as mylord and can order his men to take positions, or think about the consequences for his accompanying soldiers.

I agree.

---

Yeah, but this case is the other event. I left the m'lord untouched.

> But this plague will bring destruction to the rest of my people. My actions will have dire consequences, not only for the citizens but my followers as well..."
 
These "my" I think should be replaced with "our" as the vassal doesn't make the choice, Arthas does.